### PR TITLE
go 1.9.2

### PR DESCRIPTION
**Pull request by mblair**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by mblair** [comment]
Not sure what originally happened with the title, both `brew bump-formula-pr` and [hub](https://github.com/github/hub) freaked out.

File: Formula/go.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.9.1.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.9.1.src.tar.gz"
-    sha256 "a84afc9dc7d64fe0fa84d4d735e2ece23831a22117b50dafc75c1484f1cb550e"
+    url "https://storage.googleapis.com/golang/go1.9.2.src.tar.gz"
+    version "1.9.2"
```
Comment (DomT4) [inline Formula/go.rb]: It shouldn't be necessary to re-add this line?

Comment (mblair) [reply Formula/go.rb]: That's cool if not! I just looked in my shell history for the last `brew bump-formula-pr` that I performed, and it had `--version=`, so I used that. I'll remove this line if it's not needed.

Comment (DomT4) [reply Formula/go.rb]: It should be okay now. _Should_. Version detection on the `brew` side of things was improved.

Comment (mblair) [reply Formula/go.rb]: Swee, removed. I added another commit, can't remember if amending existing commits is cool or not.

Comment (DomT4) [reply Formula/go.rb]: Everything gets squashed down to one commit per formula changed prior to merge, generally. Sometimes Jenkins can throw a tantrum about force pushes but yeah that's the standard workflow.

**Comment by DomT4** [comment]
Oh, can you revision bump `godep` as well please? That should flip this PR to green.

File: Formula/go.rb [hunk 2]
```diff
@@ -3,9 +3,8 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.9.1.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.9.1.src.tar.gz"
```
Comment (cblecker) [inline Formula/go.rb]: Why are we removing the mirror?

Comment (DomT4) [reply Formula/go.rb]: They haven't updated it yet.

Comment (ilovezfs) [reply Formula/go.rb]: It doesn't matter if they've updated it or not.

Comment (DomT4) [reply Formula/go.rb]: I thought there was an audit for invalid mirrors now? Did I imagine that?

Comment (ilovezfs) [reply Formula/go.rb]: It's --strict precisely due to this situation.

**Comment by ilovezfs** [comment]
Thanks @mblair! :sparkle:

File: Formula/go.rb (hunk 2 above)
Comment (DomT4) [reply Formula/go.rb]: Ah, yeah, I just run `--strict` on everything which explains why I hadn't really noticed the difference 😅.

Comment (mblair) [reply Formula/go.rb]: Ah, TIL about `--strict`! I'll use that next time 😄

Comment (ilovezfs) [reply Formula/go.rb]: And voilà like clockwork the mirror is valid today. :tada:

**Pull request closed without merging**